### DISCUSSION TITLE
DIA-NN crashes without giving a specific error

### DISCUSSION THREAD

**Discussion by luciadivagno** [vdemichev/DiaNN#940] [Q&A]
Hi, I keep having problems with DIA-NN crashing without an obvious reason. Could you please help me understand why this is happening?

diann.exe --f "E:\Lucia\3347\R1_P1_S3-A1_1_30553.d
" --f "E:\Lucia\3347\R1_P2_S3-A2_1_30554.d
" --f "E:\Lucia\3347\R1_P3_S3-A3_1_30555.d
" --f "E:\Lucia\3347\R1_P4_S3-A4_1_30556.d
" --f "E:\Lucia\3347\R1_P5_S3-A5_1_30557.d
" --f "E:\Lucia\3347\R1_P6_S3-A6_1_30558.d
" --f "E:\Lucia\3347\R1_P7_S3-A7_1_30559.d
" --f "E:\Lucia\3347\R1_P8_S3-A8_1_30560.d
" --f "E:\Lucia\3347\R2_P1_S3-B1_1_30561.d
" --f "E:\Lucia\3347\R2_P2_S3-B2_1_30562.d
" --f "E:\Lucia\3347\R2_P3_S3-B3_1_30563.d
" --f "E:\Lucia\3347\R2_P4_S3-B4_1_30564.d
" --f "E:\Lucia\3347\R2_P5_S3-B5_1_30565.d
" --f "E:\Lucia\3347\R2_P6_S3-B6_1_30566.d
" --f "E:\Lucia\3347\R2_P7_S3-B7_1_30567.d
" --lib "" --threads 7 --verbose 1 --out "E:\Lucia\3347\Output\report.tsv" --qvalue 0.01 --matrices  --temp "E:\Lucia\3347\Output" --out-lib "E:\Lucia\3347\Output\lib.tsv" --gen-spec-lib --predictor --fasta "E:\fasta files\homo_sapiens.fasta" --fasta-search --min-fr-mz 200 --max-fr-mz 1800 --met-excision --cut K*,R* --missed-cleavages 1 --min-pep-len 7 --max-pep-len 30 --min-pr-mz 300 --max-pr-mz 1800 --min-pr-charge 1 --max-pr-charge 4 --unimod4 --reanalyse --smart-profiling --peak-center --no-ifs-removal 
DIA-NN 1.8.1 (Data-Independent Acquisition by Neural Networks)
Compiled on Apr 14 2022 15:31:19
Current date and time: Thu Feb 15 16:44:39 2024
CPU: GenuineIntel Intel Core Processor (Haswell, no TSX)
SIMD instructions: AVX AVX2 FMA SSE4.1 SSE4.2 
Logical CPU cores: 8
Thread number set to 7
Output will be filtered at 0.01 FDR
Precursor/protein x samples expression level matrices will be saved along with the main report
A spectral library will be generated
Deep learning will be used to generate a new in silico spectral library from peptides provided
Library-free search enabled
Min fragment m/z set to 200
Max fragment m/z set to 1800
N-terminal methionine excision enabled
In silico digest will involve cuts at K*,R*
Maximum number of missed cleavages set to 1
Min peptide length set to 7
Max peptide length set to 30
Min precursor m/z set to 300
Max precursor m/z set to 1800
Min precursor charge set to 1
Max precursor charge set to 4
Cysteine carbamidomethylation enabled as a fixed modification
A spectral library will be created from the DIA runs and used to reanalyse them; .quant files will only be saved to disk during the first step
When generating a spectral library, in silico predicted spectra will be retained if deemed more reliable than experimental ones
Fixed-width center of each elution peak will be used for quantification
Interference removal from fragment elution curves disabled
DIA-NN will optimise the mass accuracy automatically using the first run in the experiment. This is useful primarily for quick initial analyses, when it is not yet known which mass accuracy setting works best for a particular acquisition scheme.
Exclusion of fragments shared between heavy and light peptides from quantification is not supported in FASTA digest mode - disabled; to enable, generate an in silico predicted spectral library and analyse with this library

15 files will be processed
[0:00] Loading FASTA E:\fasta files\homo_sapiens.fasta
[0:06] Processing FASTA
[0:21] Assembling elution groups
[0:30] 4297569 precursors generated
[0:30] Gene names missing for some isoforms
[0:30] Library contains 20570 proteins, and 20319 genes
[0:30] Encoding peptides for spectra and RTs prediction
[0:44] Predicting spectra and IMs
[50:02] Predicting RTs
[58:00] Decoding predicted spectra and IMs
[58:24] Decoding RTs
[58:33] Saving the library to E:\Lucia\3347\Output\lib.predicted.speclib
[58:43] Initialising library

[58:48] First pass: generating a spectral library from DIA data
[58:48] File #1/15
[58:48] Loading run E:\Lucia\3347\R1_P1_S3-A1_1_30553.d
For most diaPASEF datasets it is better to manually fix both the MS1 and MS2 mass accuracies to values in the range 10-15 ppm.

DIA-NN exited
DIA-NN-plotter.exe "E:\Lucia\3347\Output\report.stats.tsv" "E:\Lucia\3347\Output\report.tsv" "E:\Lucia\3347\Output\report.pdf"
PDF report will be generated in the background

<img width="960" alt="Capture" src="https://github.com/vdemichev/DiaNN/assets/108478371/dde3b746-311c-4b2c-a38e-151b406552f1">

**Comment by Cllozano** [comment]
Hi Lucia, 

Last time I had similar issue was because my computer was short on RAM. 

How many gb do you have ? Did you during the analysis how much it was using ?
